### DISCUSSION TITLE
Add option to show operationId

### DISCUSSION THREAD

**Discussion by ninanator** [scalar/scalar#1209] [Ideas]
# Feature Request

I would like to be able to show the [operationId](https://spec.openapis.org/oas/v3.1.0#operationObject) in the UI. This will allow developers who use auto-generated clients to be able to quickly reference the operationId, since that it is how they know the endpoint. Since this is more of a niche request, I think it would be fair to make this an optional value that can be toggled at the client level.

### Swagger UI

<img width="500" alt="Located on right side of entry" src="https://github.com/scalar/scalar/assets/17285719/e83c0983-5995-431d-950a-ef6b23298fed">

See: https://petstore.swagger.io/?displayOperationId=true

### Redocly

<img width="500" alt="Located in the URL" src="https://github.com/scalar/scalar/assets/17285719/77c1003d-00d8-43d3-8494-522d19769055">

See: https://redocly.com/docs/openapi/petstore/pet/uploadFile/

### Readme.io

<img width="500" alt="Located in the URL" src="https://github.com/scalar/scalar/assets/17285719/96b08e75-4f43-44e5-886f-5c7f95754487">

See: https://pet-store-api-explorer.readme.io/reference/uploadfile

## Proposed Positioning

<img width="500" alt="Screenshot 2024-03-15 at 6 32 55 PM" src="https://github.com/scalar/scalar/assets/17285719/b000aabb-2ab1-45d3-a70f-bf2ed3e5b6fa">

I think, based on the other elements already on the page, that this location would be the least disruptive. It could be above the `description` value and potentially italicized.

Thank you!

**Comment by marclave** [comment]
@ninanator sorry about the delay here! we can add this as a configuration option :) 

issue moved here! https://github.com/scalar/scalar/issues/1274
